docker/Dockerfile.base

```diff
@@ -13,6 +13,9 @@ RUN apt-get update -qq && \
   libpq-dev \
   # required for health check
   curl \
+  # required for some Python package builds with Ubuntu 22.04
```

Comment:
I also came across the need to install a Rust compiler for running `make install-full` in 3.10 venvs and added this [note](https://rasa.com/docs/rasa/next/installation/installing-rasa-open-source#building-from-source) for macOS users. Should we add an extra note that this could apply to Linux users too building from source?

Reply:
Yeah, it can't hurt honestly. Cryptography > 34.x and tokenizers both require Rust compiler, thats why I had to add rustc and cargo to the base image, because then wheel also won't work. This is also something that will need to be done on M1 machines if its built from source.

Reply:
Great, do you want to add/edit directly in the markdown docs file?